powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/entity/OperationTemplateEntity.java

```diff
@@ -0,0 +1,236 @@
+/*
+ * PowerAuth Server and related software components
+ * Copyright (C) 2020 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package io.getlime.security.powerauth.app.server.database.model.entity;
+
+import io.getlime.security.powerauth.app.server.database.model.SignatureTypeConverter;
+import io.getlime.security.powerauth.crypto.lib.enums.PowerAuthSignatureTypes;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.Objects;
+
+/**
+ * Entity representing an operation template.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+@Entity
+@Table(name = "pa_operation_template")
+public class OperationTemplateEntity implements Serializable {
```

Comment:
Can we add `nullable=false` for values which are mandatory?

Reply:
Yes.